plugins/janus_sipre.c

```diff
@@ -936,6 +939,32 @@ static void janus_sipre_random_string(int length, char *buffer) {
 	}
 }
 
+static void parse_custom_headers(json_t *root, char *custom_headers) {
```

Comment:
Same here for the prefix, but with `janus_sipre_`.

Reply:
Done.